### PR TITLE
Fixing Errors when building documentation via `tox` commands without Notebooks 

### LINKED ISSUE #1186: Cannot build documentation and skip notebooks
If I try `POLIASTRO_SKIP_NOTEBOOKS=True tox -e docs -v`, I get errors like

```
Traceback (most recent call last):                                                                                     
  File "/home/juanlu/Projects/LSF/poliastro/library/.tox/docs/lib/python3.8/site-packages/sphinx/cmd/build.py", line 280, in build_main
    app.build(args.force_all, filenames)                                                                               
  File "/home/juanlu/Projects/LSF/poliastro/library/.tox/docs/lib/python3.8/site-packages/sphinx/application.py", line 350, in build
    self.builder.build_update()
  File "/home/juanlu/Projects/LSF/poliastro/library/.tox/docs/lib/python3.8/site-packages/sphinx/builders/__init__.py", line 292, in build_update
    self.build(to_build,            
  File "/home/juanlu/Projects/LSF/poliastro/library/.tox/docs/lib/python3.8/site-packages/sphinx/builders/__init__.py", line 306, in build
    updated_docnames = set(self.read())
  File "/usr/lib/python3.8/contextlib.py", line 120, in __exit__
    next(self.gen)                                                                                                                                                                                                                            
  File "/home/juanlu/Projects/LSF/poliastro/library/.tox/docs/lib/python3.8/site-packages/sphinx/util/logging.py", line 213, in pending_warnings
    memhandler.flushTo(logger)                                                                                                                                                                                                                
  File "/home/juanlu/Projects/LSF/poliastro/library/.tox/docs/lib/python3.8/site-packages/sphinx/util/logging.py", line 178, in flushTo
    logger.handle(record)                                                                                                                                                                                                                     
  File "/usr/lib/python3.8/logging/__init__.py", line 1587, in handle
    self.callHandlers(record)                                                                                                                                                                                                                 
  File "/usr/lib/python3.8/logging/__init__.py", line 1649, in callHandlers
    hdlr.handle(record)                                                                                                
  File "/usr/lib/python3.8/logging/__init__.py", line 946, in handle
    rv = self.filter(record)                                                                                                                                                                                                                  
  File "/usr/lib/python3.8/logging/__init__.py", line 807, in filter
    result = f.filter(record)                                                                                                                                                                                                                 
  File "/home/juanlu/Projects/LSF/poliastro/library/.tox/docs/lib/python3.8/site-packages/sphinx/util/logging.py", line 423, in filter
    raise exc                                                                                                          
sphinx.errors.SphinxWarning: /home/juanlu/Projects/LSF/poliastro/library/docs/source/examples/Analyzing NEOs.mystnb:"nbsphinx-thumbnail": No outputs in cell 6
                                                                                                                       
Warning, treated as error:
/home/juanlu/Projects/LSF/poliastro/library/docs/source/examples/Analyzing NEOs.mystnb:"nbsphinx-thumbnail": No outputs in cell 6
```

So we should either:

- Find out if we are misusing nbsphinx,
- Or whether this is a bug upstream,
- Or wether we should remove the option to skip the notebooks.

### PR DESCRIPTION
**Pull request by DhruvSondhi**
Hello, 
This PR aims to Fix #1186 
This PR implements hardcoded `nbsphinx thumbnails` which successfully fixes the non-generation of the thumbnails for the notebooks.
Thanks :)

### CODE REVIEW

File: docs/source/examples/Analyzing the Parker Solar Probe flybys.mystnb [hunk 1]
```diff
@@ -265,5 +264,9 @@ frame = StaticOrbitPlotter(plane=Planes.EARTH_ECLIPTIC)
 frame.plot_body_orbit(Earth, d_launch)
 frame.plot_body_orbit(Venus, flyby_1_time)
 frame.plot(ss01, label="#0 to #1", color="C2")
-frame.plot(ss_1_a, label="#1 to #2", color="C3");
+frame.plot(ss_1_a, label="#1 to #2", color="C3")
+```
+
+```{code-cell} ipython3
```
Comment (DhruvSondhi) [inline docs/source/examples/Analyzing the Parker Solar Probe flybys.mystnb:270]: Wrong Cell Addition

File: docs/source/examples/Multirevolutions solution in Lamberts problem.mystnb [hunk 2]
```diff
@@ -184,4 +182,6 @@ op.plot_body_orbit(Mars, EPOCH_ARR)
 for ss in orbit_set:
     lines = op.plot(ss, label=orbit_set[ss][0], color=orbit_set[ss][1], trail=True)
     lines[0].set_linestyle(orbit_set[ss][-1])
+
```
Comment (DhruvSondhi) [inline docs/source/examples/Multirevolutions solution in Lamberts problem.mystnb:185]: Wrong code addition

File: docs/source/examples/Tisserand.mystnb [hunk 3]
```diff
@@ -150,4 +148,6 @@ for planet in SS_BODIES_OUTTER:
         tp.plot(planet, (1, 7) * u.km / u.s, num_contours=7)
     else:
         tp.plot(planet, (1, 5) * u.km / u.s, num_contours=10)
+
```
Comment (DhruvSondhi) [inline docs/source/examples/Tisserand.mystnb:151]: wrong addition

**Comment by DhruvSondhi** [comment]
Hey @astrojuanlu, I have made some changes to the way the notebook thumbnails are generated. Would love to have a review on the same :)

**Comment by DhruvSondhi** [comment]
Thumbnail for `Revisiting Lambert's problem` is not working ... Investigating on the same 🔍

**Comment by abdulmoiz37** [comment]
Hey @DhruvSondhi All Tests Were Occurred Correctly :),Although py_38 test is failing which I think occurred by Reopening this PR.

**Review by astrojuanlu — COMMENTED** [review]
Thanks @DhruvSondhi for trying to fix this issue! I wonder if we want to hardcode the thumbnails like this though, we lose some of the dynamism. However, it might not be a big deal.

On the other hand, the original intent is to address #1186, but if we don't fix it it's not the end of the world either.

I'm not sure what can help us make the decision. What do you think @jorgepiloto?

**Comment by mgeier** [comment]
> I wonder if we want to hardcode the thumbnails like this though, we lose some of the dynamism.

Probably the dynamism can be kept by ignoring the warnings?

See https://github.com/poliastro/poliastro/issues/1186#issuecomment-866167372.

**Comment by astrojuanlu** [comment]
Correct! I very much prefer that approach. Closing this in favor of https://github.com/poliastro/poliastro/issues/1186#issuecomment-866342753, thanks again @mgeier

**Pull request closed without merging**